CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Unreleased
-- Added new `GDTCOREvent` initializer to support attaching a product ID to an event.
+- Added new APIs to support attaching a product ID data to an event.
```

Comment:
Consider "Added new APIs for attaching a product ID data to an event.

Reply:
```suggestion
- Added new APIs for attaching product ID data to an event.
```

Reply:
Thanks, that's better.